From 53f3e576432c19ccfd3b8acaa40259136c7f60c6 Mon Sep 17 00:00:00 2001
From: WangYuli <wangyuli@uniontech.com>
Date: Sat, 28 Dec 2024 17:53:31 +0800
Subject: [PATCH 307/360] deepin: x86: config: Update
 deepin_x86_desktop_defconfig to better support Intel devices

Follow changes:
 *Enable more drivers to better suport Intel devices.
 *Change some kernel modules from 'y' to 'm'.

Signed-off-by: WangYuli <wangyuli@uniontech.com>
---
 arch/x86/configs/deepin_x86_desktop_defconfig | 232 ++++++++++++++++--
 1 file changed, 207 insertions(+), 25 deletions(-)

diff --git a/arch/x86/configs/deepin_x86_desktop_defconfig b/arch/x86/configs/deepin_x86_desktop_defconfig
index a67666f35311..03154de7fb37 100644
--- a/arch/x86/configs/deepin_x86_desktop_defconfig
+++ b/arch/x86/configs/deepin_x86_desktop_defconfig
@@ -59,10 +59,12 @@ CONFIG_KEXEC_BZIMAGE_VERIFY_SIG=y
 CONFIG_KEXEC_JUMP=y
 CONFIG_CRASH_DUMP=y
 CONFIG_SMP=y
+# CONFIG_GOLDFISH is not set
 CONFIG_X86_CPU_RESCTRL=y
 CONFIG_X86_NUMACHIP=y
 CONFIG_X86_VSMP=y
 CONFIG_X86_UV=y
+CONFIG_X86_GOLDFISH=y
 CONFIG_X86_INTEL_MID=y
 CONFIG_X86_INTEL_LPSS=y
 CONFIG_X86_AMD_PLATFORM_DEVICE=y
@@ -71,21 +73,24 @@ CONFIG_XEN=y
 CONFIG_XEN_PVH=y
 CONFIG_JAILHOUSE_GUEST=y
 CONFIG_ACRN_GUEST=y
+CONFIG_INTEL_TDX_GUEST=y
 CONFIG_PROCESSOR_SELECT=y
 CONFIG_GART_IOMMU=y
 CONFIG_MAXSMP=y
 CONFIG_X86_REROUTE_FOR_BROKEN_BOOT_IRQS=y
 CONFIG_X86_MCELOG_LEGACY=y
+CONFIG_PERF_EVENTS_INTEL_UNCORE=m
 CONFIG_PERF_EVENTS_INTEL_RAPL=m
 CONFIG_PERF_EVENTS_INTEL_CSTATE=m
 CONFIG_PERF_EVENTS_AMD_POWER=m
+CONFIG_PERF_EVENTS_AMD_UNCORE=m
 CONFIG_PERF_EVENTS_AMD_BRS=y
 CONFIG_X86_MSR=m
 CONFIG_X86_CPUID=m
 CONFIG_AMD_MEM_ENCRYPT=y
 CONFIG_NUMA=y
 CONFIG_ARCH_MEMORY_PROBE=y
-CONFIG_X86_PMEM_LEGACY=y
+CONFIG_X86_PMEM_LEGACY=m
 CONFIG_X86_CHECK_BIOS_CORRUPTION=y
 CONFIG_MTRR_SANITIZER_ENABLE_DEFAULT=1
 CONFIG_X86_INTEL_TSX_MODE_AUTO=y
@@ -737,6 +742,7 @@ CONFIG_HOTPLUG_PCI_CPCI_ZT5550=m
 CONFIG_HOTPLUG_PCI_CPCI_GENERIC=m
 CONFIG_HOTPLUG_PCI_SHPC=y
 CONFIG_VMD=m
+CONFIG_PCI_MESON=m
 CONFIG_PCIE_DW_PLAT_HOST=y
 CONFIG_PCIE_DW_PLAT_EP=y
 CONFIG_PCI_ENDPOINT=y
@@ -764,6 +770,8 @@ CONFIG_DEVTMPFS_MOUNT=y
 CONFIG_FW_LOADER_COMPRESS=y
 CONFIG_FW_LOADER_COMPRESS_ZSTD=y
 CONFIG_DEBUG_DEVRES=y
+CONFIG_MHI_BUS_PCI_GENERIC=m
+CONFIG_MHI_BUS_EP=m
 CONFIG_CONNECTOR=y
 CONFIG_EDD=y
 CONFIG_EDD_OFF=y
@@ -776,6 +784,7 @@ CONFIG_EFI_BOOTLOADER_CONTROL=m
 CONFIG_EFI_CAPSULE_LOADER=m
 CONFIG_RESET_ATTACK_MITIGATION=y
 CONFIG_EFI_RCI2_TABLE=y
+CONFIG_EFI_COCO_SECRET=y
 CONFIG_GNSS=m
 CONFIG_GNSS_MTK_SERIAL=m
 CONFIG_GNSS_SIRF_SERIAL=m
@@ -825,10 +834,12 @@ CONFIG_MTD_BLOCK2MTD=m
 CONFIG_MTD_ONENAND=m
 CONFIG_MTD_ONENAND_VERIFY_WRITE=y
 CONFIG_MTD_ONENAND_GENERIC=m
+CONFIG_MTD_ONENAND_OTP=y
 CONFIG_MTD_ONENAND_2X_PROGRAM=y
 CONFIG_MTD_RAW_NAND=m
 CONFIG_MTD_NAND_DENALI_PCI=m
 CONFIG_MTD_NAND_CAFE=m
+CONFIG_MTD_NAND_PHYTIUM_PCI=m
 CONFIG_MTD_NAND_MXIC=m
 CONFIG_MTD_NAND_GPIO=m
 CONFIG_MTD_NAND_PLATFORM=m
@@ -900,6 +911,7 @@ CONFIG_DS1682=m
 CONFIG_VMWARE_BALLOON=m
 CONFIG_LATTICE_ECP3_CONFIG=m
 CONFIG_SRAM=y
+CONFIG_DW_XDATA_PCIE=m
 CONFIG_XILINX_SDFEC=m
 CONFIG_C2PORT=m
 CONFIG_C2PORT_DURAMAR_2150=m
@@ -912,10 +924,17 @@ CONFIG_EEPROM_EE1004=m
 CONFIG_TI_ST=m
 CONFIG_SENSORS_LIS3_I2C=m
 CONFIG_INTEL_MEI_TXE=m
+CONFIG_INTEL_MEI_GSC=m
+CONFIG_INTEL_MEI_VSC_HW=m
+CONFIG_INTEL_MEI_VSC=m
 CONFIG_INTEL_MEI_HDCP=m
+CONFIG_INTEL_MEI_PXP=m
+CONFIG_INTEL_MEI_GSC_PROXY=m
 CONFIG_VMWARE_VMCI=m
 CONFIG_GENWQE=m
 CONFIG_ECHO=m
+CONFIG_BCM_VK=m
+CONFIG_BCM_VK_TTY=y
 CONFIG_MISC_ALCOR_PCI=m
 CONFIG_MISC_RTSX_PCI=m
 CONFIG_MISC_RTSX_USB=m
@@ -923,6 +942,7 @@ CONFIG_UACCE=m
 CONFIG_PVPANIC=y
 CONFIG_PVPANIC_MMIO=m
 CONFIG_PVPANIC_PCI=m
+CONFIG_GP_PCI1XXXX=m
 CONFIG_BLK_DEV_SD=y
 CONFIG_CHR_DEV_ST=m
 CONFIG_BLK_DEV_SR=y
@@ -2033,25 +2053,33 @@ CONFIG_CDNS_I3C_MASTER=m
 CONFIG_DW_I3C_MASTER=m
 CONFIG_SPI=y
 CONFIG_SPI_ALTERA=m
+CONFIG_SPI_ALTERA_DFL=m
 CONFIG_SPI_AXI_SPI_ENGINE=m
 CONFIG_SPI_BUTTERFLY=m
 CONFIG_SPI_CADENCE=m
 CONFIG_SPI_DESIGNWARE=m
+CONFIG_SPI_DW_DMA=y
 CONFIG_SPI_DW_PCI=m
 CONFIG_SPI_DW_MMIO=m
 CONFIG_SPI_DLN2=m
 CONFIG_SPI_GPIO=m
+CONFIG_SPI_INTEL_PCI=m
 CONFIG_SPI_LM70_LLP=m
+CONFIG_SPI_MICROCHIP_CORE=m
+CONFIG_SPI_MICROCHIP_CORE_QSPI=m
 CONFIG_SPI_LANTIQ_SSC=m
 CONFIG_SPI_OC_TINY=m
+CONFIG_SPI_PCI1XXXX=m
 CONFIG_SPI_SC18IS602=m
 CONFIG_SPI_SIFIVE=m
 CONFIG_SPI_MXIC=m
 CONFIG_SPI_XCOMM=m
+CONFIG_SPI_XILINX=m
 CONFIG_SPI_ZYNQMP_GQSPI=m
 CONFIG_SPI_AMD=m
 CONFIG_SPI_MUX=m
 CONFIG_SPI_SPIDEV=m
+CONFIG_SPI_LOOPBACK_TEST=m
 CONFIG_SPI_TLE62X0=m
 CONFIG_SPI_SLAVE=y
 CONFIG_SPI_SLAVE_TIME=m
@@ -2257,6 +2285,7 @@ CONFIG_BATTERY_UG3105=m
 CONFIG_SENSORS_AAEON=m
 CONFIG_SENSORS_ABITUGURU=m
 CONFIG_SENSORS_ABITUGURU3=m
+CONFIG_SENSORS_SMPRO=m
 CONFIG_SENSORS_AD7314=m
 CONFIG_SENSORS_AD7414=m
 CONFIG_SENSORS_AD7418=m
@@ -2282,6 +2311,7 @@ CONFIG_SENSORS_K10TEMP=m
 CONFIG_SENSORS_FAM15H_POWER=m
 CONFIG_SENSORS_APPLESMC=m
 CONFIG_SENSORS_ASB100=m
+CONFIG_SENSORS_PHYTIUM=m
 CONFIG_SENSORS_ATXP1=m
 CONFIG_SENSORS_CORSAIR_CPRO=m
 CONFIG_SENSORS_CORSAIR_PSU=m
@@ -2304,6 +2334,7 @@ CONFIG_SENSORS_GL520SM=m
 CONFIG_SENSORS_G760A=m
 CONFIG_SENSORS_G762=m
 CONFIG_SENSORS_HIH6130=m
+CONFIG_SENSORS_HS3001=m
 CONFIG_SENSORS_IBMAEM=m
 CONFIG_SENSORS_IBMPEX=m
 CONFIG_SENSORS_IIO_HWMON=m
@@ -2333,12 +2364,14 @@ CONFIG_SENSORS_MAX197=m
 CONFIG_SENSORS_MAX31722=m
 CONFIG_SENSORS_MAX31730=m
 CONFIG_SENSORS_MAX31760=m
+CONFIG_MAX31827=m
 CONFIG_SENSORS_MAX6620=m
 CONFIG_SENSORS_MAX6621=m
 CONFIG_SENSORS_MAX6639=m
 CONFIG_SENSORS_MAX6650=m
 CONFIG_SENSORS_MAX6697=m
 CONFIG_SENSORS_MAX31790=m
+CONFIG_SENSORS_MC34VR500=m
 CONFIG_SENSORS_MCP3021=m
 CONFIG_SENSORS_MLXREG_FAN=m
 CONFIG_SENSORS_TC654=m
@@ -2368,11 +2401,14 @@ CONFIG_SENSORS_NTC_THERMISTOR=m
 CONFIG_SENSORS_NCT6683=m
 CONFIG_SENSORS_NCT6775=m
 CONFIG_SENSORS_NCT6775_I2C=m
+CONFIG_SENSORS_NCT7511=m
 CONFIG_SENSORS_NCT7802=m
 CONFIG_SENSORS_NCT7904=m
 CONFIG_SENSORS_NPCM7XX=m
 CONFIG_SENSORS_NZXT_KRAKEN2=m
 CONFIG_SENSORS_NZXT_SMART2=m
+CONFIG_SENSORS_OCC_P8_I2C=m
+CONFIG_SENSORS_OXP=m
 CONFIG_SENSORS_PCF8591=m
 CONFIG_PMBUS=m
 CONFIG_SENSORS_ADM1266=m
@@ -2471,6 +2507,7 @@ CONFIG_SENSORS_W83791D=m
 CONFIG_SENSORS_W83792D=m
 CONFIG_SENSORS_W83793=m
 CONFIG_SENSORS_W83795=m
+CONFIG_SENSORS_W83795_FANCTRL=y
 CONFIG_SENSORS_W83L785TS=m
 CONFIG_SENSORS_W83L786NG=m
 CONFIG_SENSORS_W83627HF=m
@@ -2478,12 +2515,15 @@ CONFIG_SENSORS_W83627EHF=m
 CONFIG_SENSORS_WM831X=m
 CONFIG_SENSORS_WM8350=m
 CONFIG_SENSORS_XGENE=m
+CONFIG_SENSORS_INTEL_M10_BMC_HWMON=m
 CONFIG_SENSORS_ACPI_POWER=m
 CONFIG_SENSORS_ATK0110=m
 CONFIG_SENSORS_ASUS_WMI=m
 CONFIG_SENSORS_ASUS_EC=m
+CONFIG_SENSORS_HP_WMI=m
 CONFIG_THERMAL_STATISTICS=y
 CONFIG_THERMAL_GOV_FAIR_SHARE=y
+CONFIG_THERMAL_GOV_POWER_ALLOCATOR=y
 CONFIG_DEVFREQ_THERMAL=y
 CONFIG_THERMAL_EMULATION=y
 CONFIG_INTEL_POWERCLAMP=m
@@ -2523,6 +2563,7 @@ CONFIG_MAX63XX_WATCHDOG=m
 CONFIG_RETU_WATCHDOG=m
 CONFIG_ACQUIRE_WDT=m
 CONFIG_ADVANTECH_WDT=m
+CONFIG_ADVANTECH_EC_WDT=m
 CONFIG_ALIM1535_WDT=m
 CONFIG_ALIM7101_WDT=m
 CONFIG_EBC_C384_WDT=m
@@ -2536,6 +2577,7 @@ CONFIG_IBMASR=m
 CONFIG_WAFER_WDT=m
 CONFIG_I6300ESB_WDT=m
 CONFIG_IE6XX_WDT=m
+CONFIG_INTEL_MID_WATCHDOG=m
 CONFIG_ITCO_WDT=m
 CONFIG_ITCO_VENDOR_SUPPORT=y
 CONFIG_IT8712F_WDT=m
@@ -2569,11 +2611,13 @@ CONFIG_BCMA_HOST_SOC=y
 CONFIG_BCMA_DRIVER_GMAC_CMN=y
 CONFIG_BCMA_DRIVER_GPIO=y
 CONFIG_MFD_AS3711=y
+CONFIG_MFD_SMPRO=m
 CONFIG_PMIC_ADP5520=y
 CONFIG_MFD_AAT2870_CORE=y
 CONFIG_MFD_BCM590XX=m
 CONFIG_MFD_BD9571MWV=m
 CONFIG_MFD_AXP20X_I2C=m
+CONFIG_MFD_CS42L43_I2C=m
 CONFIG_MFD_MADERA=m
 CONFIG_MFD_MADERA_I2C=m
 CONFIG_MFD_MADERA_SPI=m
@@ -2587,7 +2631,7 @@ CONFIG_MFD_DA9052_SPI=y
 CONFIG_MFD_DA9052_I2C=y
 CONFIG_MFD_DA9055=y
 CONFIG_MFD_DA9062=m
-CONFIG_MFD_DA9063=y
+CONFIG_MFD_DA9063=m
 CONFIG_MFD_DA9150=m
 CONFIG_MFD_DLN2=m
 CONFIG_MFD_MC13XXX_SPI=m
@@ -2636,7 +2680,7 @@ CONFIG_MFD_SKY81452=m
 CONFIG_MFD_LP3943=m
 CONFIG_MFD_LP8788=y
 CONFIG_MFD_TI_LMU=m
-CONFIG_MFD_PALMAS=y
+CONFIG_MFD_PALMAS=m
 CONFIG_TPS6105X=m
 CONFIG_TPS65010=m
 CONFIG_TPS6507X=m
@@ -2645,8 +2689,10 @@ CONFIG_MFD_TPS65090=y
 CONFIG_MFD_TI_LP873X=m
 CONFIG_MFD_TPS6586X=y
 CONFIG_MFD_TPS65910=y
-CONFIG_MFD_TPS65912_I2C=y
-CONFIG_MFD_TPS65912_SPI=y
+CONFIG_MFD_TPS65912_I2C=m
+CONFIG_MFD_TPS65912_SPI=m
+CONFIG_MFD_TPS6594_I2C=m
+CONFIG_MFD_TPS6594_SPI=m
 CONFIG_TWL4030_CORE=y
 CONFIG_TWL6040_CORE=y
 CONFIG_MFD_LM3533=m
@@ -2666,6 +2712,8 @@ CONFIG_MFD_WM8994=m
 CONFIG_MFD_WCD934X=m
 CONFIG_MFD_ATC260X_I2C=m
 CONFIG_RAVE_SP_CORE=m
+CONFIG_MFD_INTEL_M10_BMC_SPI=m
+CONFIG_MFD_INTEL_M10_BMC_PMCI=m
 CONFIG_REGULATOR_VIRTUAL_CONSUMER=m
 CONFIG_REGULATOR_USERSPACE_CONSUMER=m
 CONFIG_REGULATOR_88PG86X=m
@@ -2681,7 +2729,6 @@ CONFIG_REGULATOR_ATC260X=m
 CONFIG_REGULATOR_AXP20X=m
 CONFIG_REGULATOR_BCM590XX=m
 CONFIG_REGULATOR_BD9571MWV=m
-CONFIG_REGULATOR_DA903X=m
 CONFIG_REGULATOR_DA9052=m
 CONFIG_REGULATOR_DA9055=m
 CONFIG_REGULATOR_DA9062=m
@@ -2933,6 +2980,7 @@ CONFIG_VIDEO_TW5864=m
 CONFIG_VIDEO_TW68=m
 CONFIG_VIDEO_TW686X=m
 CONFIG_VIDEO_ZORAN=m
+CONFIG_VIDEO_ZORAN_DC30=y
 CONFIG_VIDEO_ZORAN_ZR36060=y
 CONFIG_VIDEO_ZORAN_BUZ=y
 CONFIG_VIDEO_ZORAN_DC10=y
@@ -2944,6 +2992,9 @@ CONFIG_VIDEO_IVTV=m
 CONFIG_VIDEO_IVTV_ALSA=m
 CONFIG_VIDEO_FB_IVTV=m
 CONFIG_VIDEO_FB_IVTV_FORCE_PAT=y
+CONFIG_VIDEO_HEXIUM_GEMINI=m
+CONFIG_VIDEO_HEXIUM_ORION=m
+CONFIG_VIDEO_MXB=m
 CONFIG_VIDEO_BT848=m
 CONFIG_DVB_BT8XX=m
 CONFIG_VIDEO_COBALT=m
@@ -2974,7 +3025,13 @@ CONFIG_DVB_PLUTO2=m
 CONFIG_DVB_PT1=m
 CONFIG_DVB_PT3=m
 CONFIG_DVB_SMIPCIE=m
+CONFIG_DVB_BUDGET_CORE=m
+CONFIG_DVB_BUDGET=m
+CONFIG_DVB_BUDGET_CI=m
+CONFIG_DVB_BUDGET_AV=m
 CONFIG_VIDEO_IPU3_CIO2=m
+CONFIG_INTEL_VSC=m
+CONFIG_IPU_BRIDGE=m
 CONFIG_RADIO_MAXIRADIO=m
 CONFIG_RADIO_SAA7706H=m
 CONFIG_RADIO_SHARK=m
@@ -3065,9 +3122,6 @@ CONFIG_VIDEO_ADP1653=m
 CONFIG_VIDEO_LM3560=m
 CONFIG_VIDEO_LM3646=m
 CONFIG_VIDEO_TDA1997X=m
-CONFIG_VIDEO_TDA9840=m
-CONFIG_VIDEO_TEA6415C=m
-CONFIG_VIDEO_TEA6420=m
 CONFIG_VIDEO_TLV320AIC23B=m
 CONFIG_VIDEO_ADV7180=m
 CONFIG_VIDEO_ADV7183=m
@@ -3079,7 +3133,6 @@ CONFIG_VIDEO_TC358743_CEC=y
 CONFIG_VIDEO_TVP514X=m
 CONFIG_VIDEO_TVP7002=m
 CONFIG_VIDEO_TW9910=m
-CONFIG_VIDEO_VPX3220=m
 CONFIG_VIDEO_ADV7343=m
 CONFIG_VIDEO_ADV7393=m
 CONFIG_VIDEO_ADV7511_CEC=y
@@ -3091,11 +3144,7 @@ CONFIG_VIDEO_ST_MIPID02=m
 CONFIG_VIDEO_THS7303=m
 CONFIG_CXD2880_SPI_DRV=m
 CONFIG_VIDEO_GS1662=m
-CONFIG_DVB_TDA8261=m
-CONFIG_DVB_TUA6100=m
-CONFIG_DVB_VES1X93=m
 CONFIG_DVB_DIB9000=m
-CONFIG_DVB_L64781=m
 CONFIG_DVB_S5H1432=m
 CONFIG_DVB_CXD2880=m
 CONFIG_DVB_MN88443X=m
@@ -3428,7 +3477,10 @@ CONFIG_SND_SOC_FSL_SPDIF=m
 CONFIG_SND_SOC_FSL_ESAI=m
 CONFIG_SND_SOC_FSL_MICFIL=m
 CONFIG_SND_SOC_FSL_EASRC=m
+CONFIG_SND_SOC_FSL_XCVR=m
+CONFIG_SND_SOC_FSL_RPMSG=m
 CONFIG_SND_SOC_IMX_AUDMUX=m
+CONFIG_SND_SOC_CHV3_I2S=m
 CONFIG_SND_I2S_HI6210_I2S=m
 CONFIG_SND_SOC_IMG=y
 CONFIG_SND_SOC_IMG_I2S_IN=m
@@ -3439,9 +3491,14 @@ CONFIG_SND_SOC_IMG_SPDIF_OUT=m
 CONFIG_SND_SOC_IMG_PISTACHIO_INTERNAL_DAC=m
 CONFIG_SND_SOC_INTEL_CATPT=m
 CONFIG_SND_SST_ATOM_HIFI2_PLATFORM_PCI=m
+CONFIG_SND_SOC_INTEL_SKYLAKE=m
+CONFIG_SND_SOC_INTEL_CML_H=m
+CONFIG_SND_SOC_INTEL_CML_LP=m
+CONFIG_SND_SOC_INTEL_SKYLAKE_HDAUDIO_CODEC=y
 CONFIG_SND_SOC_INTEL_AVS=m
 CONFIG_SND_SOC_INTEL_AVS_MACH_DA7219=m
 CONFIG_SND_SOC_INTEL_AVS_MACH_DMIC=m
+CONFIG_SND_SOC_INTEL_AVS_MACH_ES8336=m
 CONFIG_SND_SOC_INTEL_AVS_MACH_HDAUDIO=m
 CONFIG_SND_SOC_INTEL_AVS_MACH_MAX98927=m
 CONFIG_SND_SOC_INTEL_AVS_MACH_MAX98357A=m
@@ -3451,6 +3508,7 @@ CONFIG_SND_SOC_INTEL_AVS_MACH_PROBE=m
 CONFIG_SND_SOC_INTEL_AVS_MACH_RT274=m
 CONFIG_SND_SOC_INTEL_AVS_MACH_RT286=m
 CONFIG_SND_SOC_INTEL_AVS_MACH_RT298=m
+CONFIG_SND_SOC_INTEL_AVS_MACH_RT5663=m
 CONFIG_SND_SOC_INTEL_AVS_MACH_RT5682=m
 CONFIG_SND_SOC_INTEL_AVS_MACH_SSM4567=m
 CONFIG_SND_SOC_INTEL_USER_FRIENDLY_LONG_NAMES=y
@@ -3464,6 +3522,12 @@ CONFIG_SND_SOC_INTEL_CHT_BSW_NAU8824_MACH=m
 CONFIG_SND_SOC_INTEL_BYT_CHT_CX2072X_MACH=m
 CONFIG_SND_SOC_INTEL_BYT_CHT_DA7213_MACH=m
 CONFIG_SND_SOC_INTEL_BYT_CHT_ES8316_MACH=m
+CONFIG_SND_SOC_INTEL_BYT_CHT_NOCODEC_MACH=m
+CONFIG_SND_SOC_INTEL_SKL_RT286_MACH=m
+CONFIG_SND_SOC_INTEL_SKL_NAU88L25_SSM4567_MACH=m
+CONFIG_SND_SOC_INTEL_SKL_NAU88L25_MAX98357A_MACH=m
+CONFIG_SND_SOC_INTEL_BXT_DA7219_MAX98357A_MACH=m
+CONFIG_SND_SOC_INTEL_BXT_RT298_MACH=m
 CONFIG_SND_SOC_INTEL_SOF_WM8804_MACH=m
 CONFIG_SND_SOC_INTEL_GLK_DA7219_MAX98357A_MACH=m
 CONFIG_SND_SOC_INTEL_GLK_RT5682_MAX98357A_MACH=m
@@ -3474,6 +3538,7 @@ CONFIG_SND_SOC_INTEL_SOF_ES8336_MACH=m
 CONFIG_SND_SOC_INTEL_SOF_NAU8825_MACH=m
 CONFIG_SND_SOC_INTEL_CML_LP_DA7219_MAX98357A_MACH=m
 CONFIG_SND_SOC_INTEL_SOF_CML_RT1011_RT5682_MACH=m
+CONFIG_SND_SOC_INTEL_SOF_DA7219_MACH=m
 CONFIG_SND_SOC_INTEL_SOF_SSP_AMP_MACH=m
 CONFIG_SND_SOC_INTEL_EHL_RT5660_MACH=m
 CONFIG_SND_SOC_INTEL_SOUNDWIRE_SOF_MACH=m
@@ -3483,6 +3548,7 @@ CONFIG_SND_SOC_SOF_PCI=m
 CONFIG_SND_SOC_SOF_ACPI=m
 CONFIG_SND_SOC_SOF_AMD_TOPLEVEL=m
 CONFIG_SND_SOC_SOF_AMD_RENOIR=m
+CONFIG_SND_SOC_SOF_AMD_VANGOGH=m
 CONFIG_SND_SOC_SOF_AMD_REMBRANDT=m
 CONFIG_SND_SOC_SOF_INTEL_TOPLEVEL=y
 CONFIG_SND_SOC_SOF_HDA_LINK=y
@@ -3509,9 +3575,13 @@ CONFIG_SND_SOC_AK4642=m
 CONFIG_SND_SOC_AK5386=m
 CONFIG_SND_SOC_AK5558=m
 CONFIG_SND_SOC_ALC5623=m
+CONFIG_SND_SOC_AUDIO_IIO_AUX=m
 CONFIG_SND_SOC_AW8738=m
+CONFIG_SND_SOC_AW88395=m
+CONFIG_SND_SOC_AW88261=m
 CONFIG_SND_SOC_BD28623=m
 CONFIG_SND_SOC_BT_SCO=m
+CONFIG_SND_SOC_CHV3_CODEC=m
 CONFIG_SND_SOC_CS35L32=m
 CONFIG_SND_SOC_CS35L33=m
 CONFIG_SND_SOC_CS35L34=m
@@ -3519,6 +3589,8 @@ CONFIG_SND_SOC_CS35L35=m
 CONFIG_SND_SOC_CS35L36=m
 CONFIG_SND_SOC_CS35L45_SPI=m
 CONFIG_SND_SOC_CS35L45_I2C=m
+CONFIG_SND_SOC_CS35L56_I2C=m
+CONFIG_SND_SOC_CS35L56_SPI=m
 CONFIG_SND_SOC_CS42L51_I2C=m
 CONFIG_SND_SOC_CS42L52=m
 CONFIG_SND_SOC_CS42L56=m
@@ -3538,8 +3610,12 @@ CONFIG_SND_SOC_ES7134=m
 CONFIG_SND_SOC_ES7241=m
 CONFIG_SND_SOC_ES8328_I2C=m
 CONFIG_SND_SOC_ES8328_SPI=m
+CONFIG_SND_SOC_ES8336=m
+CONFIG_SND_SOC_ES8388=m
 CONFIG_SND_SOC_GTM601=m
 CONFIG_SND_SOC_ICS43432=m
+CONFIG_SND_SOC_IDT821034=m
+CONFIG_SND_SOC_INNO_RK3036=m
 CONFIG_SND_SOC_MAX98088=m
 CONFIG_SND_SOC_MAX98504=m
 CONFIG_SND_SOC_MAX9867=m
@@ -3560,13 +3636,18 @@ CONFIG_SND_SOC_PCM3168A_I2C=m
 CONFIG_SND_SOC_PCM3168A_SPI=m
 CONFIG_SND_SOC_PCM5102A=m
 CONFIG_SND_SOC_PCM512x_SPI=m
+CONFIG_SND_SOC_PEB2466=m
+CONFIG_SND_SOC_RK3328=m
+CONFIG_SND_SOC_RT1017_SDCA_SDW=m
 CONFIG_SND_SOC_RT5616=m
 CONFIG_SND_SOC_RT5631=m
 CONFIG_SND_SOC_RT5659=m
+CONFIG_SND_SOC_RT722_SDCA_SDW=m
 CONFIG_SND_SOC_RT9120=m
 CONFIG_SND_SOC_SGTL5000=m
 CONFIG_SND_SOC_SIMPLE_AMPLIFIER=m
 CONFIG_SND_SOC_SIMPLE_MUX=m
+CONFIG_SND_SOC_SMA1303=m
 CONFIG_SND_SOC_SPDIF=m
 CONFIG_SND_SOC_SRC4XXX_I2C=m
 CONFIG_SND_SOC_SSM2305=m
@@ -3581,6 +3662,7 @@ CONFIG_SND_SOC_TAS2562=m
 CONFIG_SND_SOC_TAS2764=m
 CONFIG_SND_SOC_TAS2770=m
 CONFIG_SND_SOC_TAS2780=m
+CONFIG_SND_SOC_TAS2781_I2C=m
 CONFIG_SND_SOC_TAS5086=m
 CONFIG_SND_SOC_TAS571X=m
 CONFIG_SND_SOC_TAS5720=m
@@ -3631,6 +3713,7 @@ CONFIG_SND_SOC_WM8978=m
 CONFIG_SND_SOC_WM8985=m
 CONFIG_SND_SOC_WSA881X=m
 CONFIG_SND_SOC_WSA883X=m
+CONFIG_SND_SOC_WSA884X=m
 CONFIG_SND_SOC_ZL38060=m
 CONFIG_SND_SOC_MAX9759=m
 CONFIG_SND_SOC_MT6351=m
@@ -4252,12 +4335,20 @@ CONFIG_DMADEVICES=y
 CONFIG_ALTERA_MSGDMA=m
 CONFIG_INTEL_IDMA64=m
 CONFIG_INTEL_IDXD=m
+CONFIG_INTEL_IDXD_SVM=y
+CONFIG_INTEL_IDXD_PERFMON=y
 CONFIG_INTEL_IOATDMA=m
 CONFIG_PLX_DMA=m
+CONFIG_XILINX_DMA=m
+CONFIG_XILINX_XDMA=m
+CONFIG_AMD_PTDMA=m
 CONFIG_QCOM_HIDMA_MGMT=m
 CONFIG_QCOM_HIDMA=m
 CONFIG_DW_DMAC=m
+CONFIG_DW_EDMA=m
+CONFIG_DW_EDMA_PCIE=m
 CONFIG_SF_PDMA=m
+CONFIG_INTEL_LDMA=y
 CONFIG_ASYNC_TX_DMA=y
 CONFIG_SW_SYNC=y
 CONFIG_UDMABUF=y
@@ -4441,6 +4532,8 @@ CONFIG_AD9834=m
 CONFIG_AD5933=m
 CONFIG_FB_SM750=m
 CONFIG_STAGING_MEDIA=y
+CONFIG_INTEL_ATOMISP=y
+CONFIG_DVB_AV7110=m
 CONFIG_VIDEO_IPU3_IMGU=m
 CONFIG_LTE_GDM724X=m
 CONFIG_FB_TFT=m
@@ -4538,6 +4631,7 @@ CONFIG_UV_SYSFS=m
 CONFIG_NVIDIA_WMI_EC_BACKLIGHT=m
 CONFIG_XIAOMI_WMI=m
 CONFIG_GIGABYTE_WMI=m
+CONFIG_YOGABOOK=m
 CONFIG_ACERHDF=m
 CONFIG_ACER_WIRELESS=m
 CONFIG_ACER_WMI=m
@@ -4583,6 +4677,8 @@ CONFIG_INTEL_VBTN=m
 CONFIG_INTEL_INT0002_VGPIO=m
 CONFIG_INTEL_OAKTRAIL=m
 CONFIG_INTEL_BXTWC_PMIC_TMU=m
+CONFIG_INTEL_BYTCRC_PWRSRC=m
+CONFIG_INTEL_CHTDC_TI_PWRBTN=m
 CONFIG_INTEL_CHTWC_INT33FE=m
 CONFIG_INTEL_ISHTP_ECLITE=m
 CONFIG_INTEL_MRFLD_PWRBTN=m
@@ -4590,8 +4686,10 @@ CONFIG_INTEL_PUNIT_IPC=m
 CONFIG_INTEL_RST=m
 CONFIG_INTEL_SDSI=m
 CONFIG_INTEL_SMARTCONNECT=m
+CONFIG_INTEL_TPMI=m
 CONFIG_INTEL_TURBO_MAX_3=y
 CONFIG_INTEL_VSEC=m
+CONFIG_MSI_EC=m
 CONFIG_MSI_LAPTOP=m
 CONFIG_MSI_WMI=m
 CONFIG_PCENGINES_APU2=m
@@ -4619,6 +4717,7 @@ CONFIG_INTEL_SCU_PLATFORM=m
 CONFIG_INTEL_SCU_IPC_UTIL=m
 CONFIG_SIEMENS_SIMATIC_IPC=m
 CONFIG_WINMATE_FM07_KEYS=m
+CONFIG_SEL3350_PLATFORM=m
 CONFIG_COMMON_CLK_WM831X=m
 CONFIG_LMK04832=m
 CONFIG_COMMON_CLK_MAX9485=m
@@ -4640,18 +4739,20 @@ CONFIG_INTEL_IOMMU_SVM=y
 # CONFIG_INTEL_IOMMU_DEFAULT_ON is not set
 # CONFIG_INTEL_IOMMU_SCALABLE_MODE_DEFAULT_ON is not set
 CONFIG_IRQ_REMAP=y
+CONFIG_VIRTIO_IOMMU=m
 CONFIG_REMOTEPROC=y
 CONFIG_RPMSG_CHAR=m
 CONFIG_RPMSG_QCOM_GLINK_RPM=m
 CONFIG_RPMSG_VIRTIO=m
 CONFIG_SOUNDWIRE=m
+CONFIG_SOUNDWIRE_AMD=m
 CONFIG_SOUNDWIRE_QCOM=m
+CONFIG_WPCM450_SOC=m
 CONFIG_SOC_TI=y
-CONFIG_DEVFREQ_GOV_SIMPLE_ONDEMAND=y
-CONFIG_DEVFREQ_GOV_PERFORMANCE=y
-CONFIG_DEVFREQ_GOV_POWERSAVE=y
-CONFIG_DEVFREQ_GOV_USERSPACE=y
-CONFIG_DEVFREQ_GOV_PASSIVE=y
+CONFIG_DEVFREQ_GOV_PERFORMANCE=m
+CONFIG_DEVFREQ_GOV_POWERSAVE=m
+CONFIG_DEVFREQ_GOV_USERSPACE=m
+CONFIG_DEVFREQ_GOV_PASSIVE=m
 CONFIG_PM_DEVFREQ_EVENT=y
 CONFIG_EXTCON_ADC_JACK=m
 CONFIG_EXTCON_AXP288=m
@@ -4670,25 +4771,36 @@ CONFIG_EXTCON_RT8973A=m
 CONFIG_EXTCON_SM5502=m
 CONFIG_EXTCON_USB_GPIO=m
 CONFIG_EXTCON_USBC_CROS_EC=m
+CONFIG_EXTCON_USBC_TUSB320=m
 CONFIG_MEMORY=y
-CONFIG_IIO_BUFFER_DMAENGINE=m
-CONFIG_IIO_BUFFER_HW_CONSUMER=m
+CONFIG_FPGA_DFL_EMIF=m
 CONFIG_IIO_SW_DEVICE=m
 CONFIG_IIO_SW_TRIGGER=m
 CONFIG_ADIS16201=m
 CONFIG_ADIS16209=m
+CONFIG_ADXL313_I2C=m
+CONFIG_ADXL313_SPI=m
+CONFIG_ADXL355_I2C=m
+CONFIG_ADXL355_SPI=m
+CONFIG_ADXL367_SPI=m
+CONFIG_ADXL367_I2C=m
 CONFIG_ADXL372_SPI=m
 CONFIG_ADXL372_I2C=m
 CONFIG_BMA220=m
 CONFIG_BMA400=m
 CONFIG_BMC150_ACCEL=m
+CONFIG_BMI088_ACCEL=m
 CONFIG_DA280=m
 CONFIG_DA311=m
+CONFIG_DMARD06=m
 CONFIG_DMARD09=m
 CONFIG_DMARD10=m
+CONFIG_FXLS8962AF_I2C=m
+CONFIG_FXLS8962AF_SPI=m
 CONFIG_HID_SENSOR_ACCEL_3D=m
 CONFIG_IIO_CROS_EC_ACCEL_LEGACY=m
-CONFIG_IIO_ST_ACCEL_3AXIS=m
+CONFIG_IIO_KX022A_SPI=m
+CONFIG_IIO_KX022A_I2C=m
 CONFIG_KXSD9=m
 CONFIG_KXCJK1013=m
 CONFIG_MC3230=m
@@ -4698,11 +4810,14 @@ CONFIG_MMA7660=m
 CONFIG_MMA8452=m
 CONFIG_MMA9551=m
 CONFIG_MMA9553=m
+CONFIG_MSA311=m
 CONFIG_MXC4005=m
 CONFIG_MXC6255=m
 CONFIG_SCA3000=m
+CONFIG_SCA3300=m
 CONFIG_STK8312=m
 CONFIG_STK8BA50=m
+CONFIG_AD4130=m
 CONFIG_AD7091R5=m
 CONFIG_AD7124=m
 CONFIG_AD7192=m
@@ -4723,11 +4838,14 @@ CONFIG_AD7887=m
 CONFIG_AD7923=m
 CONFIG_AD7949=m
 CONFIG_AD799X=m
+CONFIG_AD9467=m
+CONFIG_ADI_AXI_ADC=m
 CONFIG_AXP20X_ADC=m
 CONFIG_AXP288_ADC=m
 CONFIG_CC10001_ADC=m
 CONFIG_DA9150_GPADC=m
 CONFIG_DLN2_ADC=m
+CONFIG_ENVELOPE_DETECTOR=m
 CONFIG_HI8435=m
 CONFIG_HX711=m
 CONFIG_INA2XX_ADC=m
@@ -4740,12 +4858,16 @@ CONFIG_LTC2497=m
 CONFIG_MAX1027=m
 CONFIG_MAX11100=m
 CONFIG_MAX1118=m
+CONFIG_MAX11205=m
+CONFIG_MAX11410=m
 CONFIG_MAX1241=m
 CONFIG_MAX1363=m
 CONFIG_MAX9611=m
 CONFIG_MCP320X=m
 CONFIG_MCP3422=m
 CONFIG_MCP3911=m
+CONFIG_MEDIATEK_MT6360_ADC=m
+CONFIG_MEDIATEK_MT6370_ADC=m
 CONFIG_MEN_Z188_ADC=m
 CONFIG_MP2629_ADC=m
 CONFIG_NAU7802=m
@@ -4753,6 +4875,8 @@ CONFIG_PALMAS_GPADC=m
 CONFIG_QCOM_SPMI_IADC=m
 CONFIG_QCOM_SPMI_VADC=m
 CONFIG_QCOM_SPMI_ADC5=m
+CONFIG_RICHTEK_RTQ6056=m
+CONFIG_SD_ADC_MODULATOR=m
 CONFIG_TI_ADC081C=m
 CONFIG_TI_ADC0832=m
 CONFIG_TI_ADC084S021=m
@@ -4761,14 +4885,27 @@ CONFIG_TI_ADC108S102=m
 CONFIG_TI_ADC128S052=m
 CONFIG_TI_ADC161S626=m
 CONFIG_TI_ADS1015=m
+CONFIG_TI_ADS7924=m
+CONFIG_TI_ADS1100=m
 CONFIG_TI_ADS7950=m
+CONFIG_TI_ADS8344=m
+CONFIG_TI_ADS8688=m
+CONFIG_TI_ADS124S08=m
+CONFIG_TI_ADS131E08=m
+CONFIG_TI_LMP92064=m
 CONFIG_TI_TLC4541=m
+CONFIG_TI_TSC2046=m
 CONFIG_TWL4030_MADC=m
 CONFIG_TWL6030_GPADC=m
+CONFIG_VF610_ADC=m
 CONFIG_VIPERBOARD_ADC=m
 CONFIG_XILINX_XADC=m
+CONFIG_AD74115=m
+CONFIG_AD74413R=m
 CONFIG_STX104=m
+CONFIG_IIO_RESCALE=m
 CONFIG_AD8366=m
+CONFIG_ADA4250=m
 CONFIG_HMC425=m
 CONFIG_AD7150=m
 CONFIG_AD7746=m
@@ -4781,13 +4918,19 @@ CONFIG_PMS7003=m
 CONFIG_SCD30_CORE=m
 CONFIG_SCD30_I2C=m
 CONFIG_SCD30_SERIAL=m
+CONFIG_SCD4X=m
 CONFIG_SENSIRION_SGP30=m
+CONFIG_SENSIRION_SGP40=m
+CONFIG_SPS30_I2C=m
+CONFIG_SPS30_SERIAL=m
+CONFIG_SENSEAIR_SUNRISE_CO2=m
 CONFIG_VZ89X=m
 CONFIG_IIO_CROS_EC_SENSORS_CORE=m
 CONFIG_IIO_CROS_EC_SENSORS=m
 CONFIG_IIO_CROS_EC_SENSORS_LID_ANGLE=m
 CONFIG_IIO_SSP_SENSORS_COMMONS=m
 CONFIG_IIO_SSP_SENSORHUB=m
+CONFIG_AD3552R=m
 CONFIG_AD5064=m
 CONFIG_AD5360=m
 CONFIG_AD5380=m
@@ -4798,32 +4941,45 @@ CONFIG_AD5592R=m
 CONFIG_AD5593R=m
 CONFIG_AD5504=m
 CONFIG_AD5624R_SPI=m
+CONFIG_LTC2688=m
 CONFIG_AD5686_SPI=m
 CONFIG_AD5696_I2C=m
 CONFIG_AD5755=m
 CONFIG_AD5758=m
 CONFIG_AD5761=m
 CONFIG_AD5764=m
+CONFIG_AD5766=m
 CONFIG_AD5770R=m
 CONFIG_AD5791=m
+CONFIG_AD7293=m
 CONFIG_AD7303=m
 CONFIG_AD8801=m
 CONFIG_CIO_DAC=m
+CONFIG_DPOT_DAC=m
 CONFIG_DS4424=m
 CONFIG_LTC1660=m
 CONFIG_LTC2632=m
 CONFIG_M62332=m
 CONFIG_MAX517=m
+CONFIG_MAX5522=m
+CONFIG_MAX5821=m
 CONFIG_MCP4725=m
+CONFIG_MCP4728=m
 CONFIG_MCP4922=m
 CONFIG_TI_DAC082S085=m
 CONFIG_TI_DAC5571=m
 CONFIG_TI_DAC7311=m
 CONFIG_TI_DAC7612=m
+CONFIG_VF610_DAC=m
 CONFIG_IIO_SIMPLE_DUMMY=m
 CONFIG_AD9523=m
 CONFIG_ADF4350=m
 CONFIG_ADF4371=m
+CONFIG_ADF4377=m
+CONFIG_ADMV1013=m
+CONFIG_ADMV1014=m
+CONFIG_ADMV4420=m
+CONFIG_ADRF6780=m
 CONFIG_ADIS16080=m
 CONFIG_ADIS16130=m
 CONFIG_ADIS16136=m
@@ -4855,6 +5011,8 @@ CONFIG_ADIS16475=m
 CONFIG_ADIS16480=m
 CONFIG_BMI160_I2C=m
 CONFIG_BMI160_SPI=m
+CONFIG_BOSCH_BNO055_SERIAL=m
+CONFIG_BOSCH_BNO055_I2C=m
 CONFIG_FXOS8700_I2C=m
 CONFIG_FXOS8700_SPI=m
 CONFIG_KMX61=m
@@ -4863,6 +5021,7 @@ CONFIG_INV_ICM42600_SPI=m
 CONFIG_INV_MPU6050_I2C=m
 CONFIG_INV_MPU6050_SPI=m
 CONFIG_IIO_ST_LSM6DSX=m
+CONFIG_IIO_ST_LSM9DS0=m
 CONFIG_ACPI_ALS=m
 CONFIG_ADJD_S311=m
 CONFIG_ADUX1020=m
@@ -4876,6 +5035,7 @@ CONFIG_BH1780=m
 CONFIG_CM32181=m
 CONFIG_CM3232=m
 CONFIG_CM3323=m
+CONFIG_CM3605=m
 CONFIG_CM36651=m
 CONFIG_IIO_CROS_EC_LIGHT_PROX=m
 CONFIG_GP2AP002=m
@@ -4887,14 +5047,18 @@ CONFIG_ISL29125=m
 CONFIG_HID_SENSOR_ALS=m
 CONFIG_HID_SENSOR_PROX=m
 CONFIG_JSA1212=m
+CONFIG_ROHM_BU27008=m
+CONFIG_ROHM_BU27034=m
 CONFIG_RPR0521=m
 CONFIG_SENSORS_LM3533=m
 CONFIG_LTR501=m
+CONFIG_LTRF216A=m
 CONFIG_LV0104CS=m
 CONFIG_MAX44000=m
 CONFIG_MAX44009=m
 CONFIG_NOA1305=m
 CONFIG_OPT3001=m
+CONFIG_OPT4001=m
 CONFIG_PA12203001=m
 CONFIG_SI1133=m
 CONFIG_SI1145=m
@@ -4904,6 +5068,7 @@ CONFIG_TCS3414=m
 CONFIG_TCS3472=m
 CONFIG_SENSORS_TSL2563=m
 CONFIG_TSL2583=m
+CONFIG_TSL2591=m
 CONFIG_TSL2772=m
 CONFIG_TSL4531=m
 CONFIG_US5182D=m
@@ -4913,17 +5078,20 @@ CONFIG_VEML6030=m
 CONFIG_VEML6070=m
 CONFIG_VL6180=m
 CONFIG_ZOPT2201=m
+CONFIG_AK8974=m
 CONFIG_AK09911=m
 CONFIG_BMC150_MAGN_I2C=m
 CONFIG_BMC150_MAGN_SPI=m
 CONFIG_MAG3110=m
 CONFIG_HID_SENSOR_MAGNETOMETER_3D=m
 CONFIG_MMC35240=m
-CONFIG_IIO_ST_MAGN_3AXIS=m
 CONFIG_SENSORS_HMC5843_I2C=m
 CONFIG_SENSORS_HMC5843_SPI=m
 CONFIG_SENSORS_RM3100_I2C=m
 CONFIG_SENSORS_RM3100_SPI=m
+CONFIG_TI_TMAG5273=m
+CONFIG_YAMAHA_YAS530=m
+CONFIG_IIO_MUX=m
 CONFIG_HID_SENSOR_INCLINOMETER_3D=m
 CONFIG_HID_SENSOR_DEVICE_ROTATION=m
 CONFIG_IIO_HRTIMER_TRIGGER=m
@@ -4931,6 +5099,8 @@ CONFIG_IIO_INTERRUPT_TRIGGER=m
 CONFIG_IIO_TIGHTLOOP_TRIGGER=m
 CONFIG_IIO_SYSFS_TRIGGER=m
 CONFIG_IQS624_POS=m
+CONFIG_HID_SENSOR_CUSTOM_INTEL_HINGE=m
+CONFIG_AD5110=m
 CONFIG_AD5272=m
 CONFIG_DS1803=m
 CONFIG_MAX5432=m
@@ -4941,6 +5111,7 @@ CONFIG_MCP4131=m
 CONFIG_MCP4531=m
 CONFIG_MCP41010=m
 CONFIG_TPL0102=m
+CONFIG_X9250=m
 CONFIG_LMP91000=m
 CONFIG_ABP060MG=m
 CONFIG_BMP280=m
@@ -4953,6 +5124,7 @@ CONFIG_ICP10100=m
 CONFIG_MPL115_I2C=m
 CONFIG_MPL115_SPI=m
 CONFIG_MPL3115=m
+CONFIG_MPRLS0025PA=m
 CONFIG_MS5611=m
 CONFIG_MS5611_I2C=m
 CONFIG_MS5611_SPI=m
@@ -4962,6 +5134,8 @@ CONFIG_T5403=m
 CONFIG_HP206C=m
 CONFIG_ZPA2326=m
 CONFIG_AS3935=m
+CONFIG_CROS_EC_MKBP_PROXIMITY=m
+CONFIG_IRSD200=m
 CONFIG_ISL29501=m
 CONFIG_LIDAR_LITE_V2=m
 CONFIG_MB1232=m
@@ -4969,6 +5143,8 @@ CONFIG_PING=m
 CONFIG_RFD77402=m
 CONFIG_SRF04=m
 CONFIG_SX9310=m
+CONFIG_SX9324=m
+CONFIG_SX9360=m
 CONFIG_SX9500=m
 CONFIG_SRF08=m
 CONFIG_VCNL3020=m
@@ -4987,6 +5163,7 @@ CONFIG_TMP007=m
 CONFIG_TMP117=m
 CONFIG_TSYS01=m
 CONFIG_TSYS02D=m
+CONFIG_MAX30208=m
 CONFIG_MAX31856=m
 CONFIG_MAX31865=m
 CONFIG_NTB=m
@@ -5005,8 +5182,8 @@ CONFIG_PWM_CROS_EC=m
 CONFIG_PWM_DWC=m
 CONFIG_PWM_IQS620A=m
 CONFIG_PWM_LP3943=m
-CONFIG_PWM_LPSS_PCI=y
-CONFIG_PWM_LPSS_PLATFORM=y
+CONFIG_PWM_LPSS_PCI=m
+CONFIG_PWM_LPSS_PLATFORM=m
 CONFIG_PWM_PCA9685=m
 CONFIG_PWM_TWL=m
 CONFIG_PWM_TWL_LED=m
@@ -5017,6 +5194,7 @@ CONFIG_RESET_SIMPLE=y
 CONFIG_RESET_TI_SYSCON=m
 CONFIG_RESET_TI_TPS380X=m
 CONFIG_USB_LGM_PHY=m
+CONFIG_PHY_CAN_TRANSCEIVER=m
 CONFIG_BCM_KONA_USB2_PHY=m
 CONFIG_PHY_PXA_28NM_HSIC=m
 CONFIG_PHY_PXA_28NM_USB2=m
@@ -5027,6 +5205,7 @@ CONFIG_PHY_SAMSUNG_USB2=m
 CONFIG_PHY_TUSB1210=m
 CONFIG_PHY_INTEL_LGM_EMMC=m
 CONFIG_INTEL_RAPL=m
+CONFIG_INTEL_RAPL_TPMI=m
 CONFIG_MCB=m
 CONFIG_MCB_PCI=m
 CONFIG_MCB_LPC=m
@@ -5034,6 +5213,7 @@ CONFIG_RAS_CEC=y
 CONFIG_USB4=m
 CONFIG_ANDROID_BINDER_IPC=y
 CONFIG_ANDROID_BINDERFS=y
+CONFIG_LIBNVDIMM=y
 CONFIG_BLK_DEV_PMEM=m
 CONFIG_DEV_DAX=m
 CONFIG_NVMEM_RAVE_SP_EEPROM=m
@@ -5076,6 +5256,8 @@ CONFIG_SIOX_BUS_GPIO=m
 CONFIG_SLIM_QCOM_CTRL=m
 CONFIG_INTERCONNECT=y
 CONFIG_104_QUAD_8=m
+CONFIG_INTEL_QEP=m
+CONFIG_INTERRUPT_CNT=m
 CONFIG_MOST=m
 CONFIG_MOST_USB_HDM=m
 CONFIG_MOST_CDEV=m
-- 
2.20.1

